<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GAPAN CITY COLLEGE Employee Managent System</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/favicon.png</normaloff>../icons/favicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="employeeinfo_toolbtn">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>10</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTip">
     <string>employee information and logs</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border: None;}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/attendance.png</normaloff>../icons/attendance.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>110</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="charts_toolbtn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTip">
     <string>charts</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border: None;}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/bargraph.png</normaloff>../icons/bargraph.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>95</width>
      <height>95</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="settings_toolbtn">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>10</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTip">
     <string>configurations</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border: None;}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/settings2.png</normaloff>../icons/settings2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>85</width>
      <height>85</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="register_toolbtn">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTip">
     <string>register employee</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border: None;}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/profile.png</normaloff>../icons/profile.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>23:45:55</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="date_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>December 29, 2018</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="table_frame">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>851</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border: None;}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="search_lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>500</y>
       <width>621</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="attendance_view">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>811</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="search_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_search">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../icons/search (1).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_close">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{border: None;}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/close.png</normaloff>../icons/close.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="user_pass">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
    <property name="gridVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="clocked_toolbtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTip">
     <string>show clocked-in employees</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border: None;}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/user_valid-512.png</normaloff>../icons/user_valid-512.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>95</width>
      <height>95</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_signin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>82</width>
      <height>100</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border: None;}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/Login-icon.png</normaloff>../icons/Login-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_signout">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>82</width>
      <height>100</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border: None;}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/Logout-icon.png</normaloff>../icons/Logout-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_logs">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>580</y>
      <width>82</width>
      <height>100</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border: None;}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/TG-Attendance-Welfare-300x300.png</normaloff>../icons/TG-Attendance-Welfare-300x300.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_alllogs">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>580</y>
      <width>82</width>
      <height>100</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border: None;}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/time.png</normaloff>../icons/time.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_help">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>580</y>
      <width>82</width>
      <height>100</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border: None;}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/helpme.png</normaloff>../icons/helpme.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="memo_toolbtn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTip">
     <string>memo</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border: None;}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/rem2.png</normaloff>../icons/rem2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>95</width>
      <height>95</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Memo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Clocked-In
Employee(s)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Graphs</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Register
Employee</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>110</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Employee's
Information
and Logs</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>120</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Configurations</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>670</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Clock-In</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>670</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Clock-Out</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>670</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>User
Logs</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>670</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>All
Logs</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>670</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>530</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../icons/forgot-password-icon-9.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>label_9</zorder>
   <zorder>label_8</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>label_7</zorder>
   <zorder>employeeinfo_toolbtn</zorder>
   <zorder>charts_toolbtn</zorder>
   <zorder>settings_toolbtn</zorder>
   <zorder>register_toolbtn</zorder>
   <zorder>time_label</zorder>
   <zorder>date_label</zorder>
   <zorder>table_frame</zorder>
   <zorder>user_pass</zorder>
   <zorder>calendarWidget</zorder>
   <zorder>clocked_toolbtn</zorder>
   <zorder>btn_signin</zorder>
   <zorder>btn_signout</zorder>
   <zorder>btn_logs</zorder>
   <zorder>btn_alllogs</zorder>
   <zorder>btn_help</zorder>
   <zorder>memo_toolbtn</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_6</zorder>
   <zorder>label_12</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>user_pass</tabstop>
  <tabstop>btn_signin</tabstop>
  <tabstop>btn_signout</tabstop>
  <tabstop>btn_logs</tabstop>
  <tabstop>btn_alllogs</tabstop>
  <tabstop>btn_help</tabstop>
  <tabstop>memo_toolbtn</tabstop>
  <tabstop>clocked_toolbtn</tabstop>
  <tabstop>charts_toolbtn</tabstop>
  <tabstop>register_toolbtn</tabstop>
  <tabstop>employeeinfo_toolbtn</tabstop>
  <tabstop>settings_toolbtn</tabstop>
  <tabstop>search_lineEdit_2</tabstop>
  <tabstop>search_lineEdit</tabstop>
  <tabstop>btn_close</tabstop>
  <tabstop>attendance_view</tabstop>
  <tabstop>calendarWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
